rllib/models/modelv2.py

```diff
@@ -314,29 +318,6 @@ def is_time_major(self) -> bool:
         """
         return self.time_major is True
 
-    # TODO: (sven) Experimental method.
-    def get_input_dict(self, sample_batch,
```

Comment:
no longer needed due to simplification in PPO's (and other algos) postprocessing function.